pkgs/games/crossfire/crossfire-server.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchsvn, autoreconfHook,
+  autoconf, automake, libtool, flex, perl, check, pkg-config, python2,
+  version, rev, sha256, maps, arch }:
+
+stdenv.mkDerivation rec {
+  version = "r${toString rev}";
+  name = "crossfire-server-${version}";
+
+  src = fetchsvn {
+    url = "http://svn.code.sf.net/p/crossfire/code/server/trunk/";
+    sha256 = sha256;
+    rev = rev;
+  };
+
+  nativeBuildInputs = [autoconf automake libtool flex perl check pkg-config python2];
```

Comment:
```suggestion
  nativeBuildInputs = [ autoconf automake libtool flex perl check pkg-config python2 ];
```
can we use python3?

Reply:
At the time I wrote this, python3 support was still in the early planning stages. It looks like may be at least partially supported now, but I'm hesitant to change too much considering how long this has been in flight; I'd rather commit it in its known-working state and then do a followup PR that updates crossfire-server-latest to 2021 and, if possible, updates the python dep to py3.